### DISCUSSION TITLE
After running pnpm i and yarn dev, encounter error that says @hoppscotch/codemirror-lang-graphql @hoppscotch/data @hoppscotch/js-sandbox not found

### DISCUSSION THREAD

**Discussion by Giancarlo-Ma** [hoppscotch/hoppscotch#2224] [Q&A] [answered]
ANY STEP I MISSED?

**Comment by Giancarlo-Ma** [comment]
![image](https://user-images.githubusercontent.com/65126107/161226717-e34ed573-d1f4-438b-b6aa-f51e90800665.png)

**Reply by Giancarlo-Ma** [reply]
WHY packages in packages not found

**Comment by AndrewBastin** [comment]
@Giancarlo-Ma can you send the output of `pnpm i` ?

**Reply by Giancarlo-Ma** [reply]
![image](https://user-images.githubusercontent.com/65126107/161229723-22530271-ae10-4564-9a35-57420e1c7b0e.png)

**Comment by AndrewBastin** [comment, marked as answer]
Aaah yes, it looks like you have installed `pnpm` as root. This leads to PNPM adding a bit of restrictions which block the build scripts from running, to avoid this, instead of using `pnpm i` you can use `pnpm i --unsafe-perm` to allow us to run the scripts which build the packages. Once that is done, you can run the other scripts as normal

**Reply by Giancarlo-Ma** [reply]
thx
